json-schema/python/frinx_api/json_schema/device_update.py

```diff
@@ -0,0 +1,101 @@
+# generated by datamodel-codegen
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import Any
+from typing import Optional
+
+from pydantic import BaseModel
+from pydantic import ConfigDict
+from pydantic import Field
+
+
+class DeviceSize(Enum):
+    """
+    The size of the device.
+    """
+
+    SMALL = 'SMALL'
+    MEDIUM = 'MEDIUM'
+    LARGE = 'LARGE'
+
+
+class ServiceState(Enum):
+    """
+    The state the device is in.
+    """
+
+    IN_SERVICE = 'IN_SERVICE'
+    OUT_OF_SERVICE = 'OUT_OF_SERVICE'
+    PLANNING = 'PLANNING'
+
+
+class DeviceUpdate(BaseModel):
+    """
+    Device update schema
+    """
+
+    model_config = ConfigDict(
+        populate_by_name=True,
+    )
+    device_name: str
```

Comment:
extract common fields from all types of messages to new class to avoid code duplication